suleyunus/itv-app
# Group resources endpoint

### What does this PR do?

Have group resources endpoints

### Description of task to be completed?

Have the following endpoints working:
```
POST api/v1/groups
```
```
GET api/v1/groups
```
```
GET api/v1/group/:groupId
```
```
UPDATE api/v1/group/:groupId
```
```
DELETE api/v1/group/:groupId
```
```
POST api/v1/group/:groupId/members
```
```
GET api/v1/group/:groupId/members
```
```
GET api/v1/group/:groupId/member/:memberId
```
```
DELETE api/v1/group/:groupId/member/:memberId
```
```
UPDATE api/v1/group/:groupId/member/:memberId
```

### How should this be manually tested?

After cloning the repo, cd into it and run `npm run start:dev`, this will spin up the node server after which you should be able to test the above tokens with postman.
